### PR TITLE
Model + controller for services

### PR DESCRIPTION
**Pull request by krummi**

### CODE REVIEW

File: priv/repo/seeds.exs [hunk 1]
```diff
@@ -31,11 +29,11 @@ company2 = Repo.insert! %Company{
 
 IO.inspect(company1)
 
-# Create some users
+# Create users
 
-key = "pu9-VNdgE8V9qZo19rlcg3KUNjpxuixg"
+key = "a"
```
Comment (kthelgason) [inline priv/repo/seeds.exs:34]: nice 🔑

File: priv/repo/seeds.exs [hunk 2]
```diff
@@ -79,3 +77,35 @@ employee3 = %Employee{}
...
+  cleanup_duration: 0,
+  pricing: "4500 kr.",
+  company_id: company1.id
+})
+|> Repo.insert!
+
+%Service{}
+|> Service.changeset(%{
+  name: "Haircut: Mohawk cut",
+  description: "Mohawk haircuts are the shit this summer!",
```
Comment (kthelgason) [inline priv/repo/seeds.exs:97]: 😂

File: test/controllers/service_controller_test.exs [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  end
+
+  test "does not create resource and renders errors when data is invalid",
+    %{conn: conn} do
+    conn = post conn, service_path(conn, :create), service: @invalid_attrs
+    assert html_response(conn, 200) =~ "New service"
+  end
+
+  test "shows chosen resource", %{conn: conn, valid_attrs: valid_attrs} do
+    service = Repo.insert! Service.changeset(%Service{}, valid_attrs)
```
Comment (kthelgason) [inline test/controllers/service_controller_test.exs:51]: perhaps move this to ex_machina as well?

File: web/models/service.ex [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  Creates a changeset based on the `model` and `params`.
+
+  If no params are provided, an invalid changeset is returned
+  with no validation performed.
+  """
+  def changeset(model, params \\ :empty) do
+    model
+    |> cast(params, @required_fields, @optional_fields)
+    |> validate_number(:duration, greater_than: 1, less_than: 1440) # 60 * 24
+    |> validate_number(:cleanup_duration, greater_than_or_equal_to: 0, less_than_or_equal_to: 60)
```
Comment (kthelgason) [inline web/models/service.ex]: line too long 👾

**Comment by kthelgason** [comment]
Few comments, otherwise LGTM :+1:

File: priv/repo/seeds.exs (hunk 1 above)
Comment (krummi) [inline priv/repo/seeds.exs:34]: Turns out this is the confirmation key and thus doesn't do shite 💩

File: test/controllers/service_controller_test.exs (hunk 3 above)
Comment (krummi) [inline test/controllers/service_controller_test.exs:51]: absolutely

File: web/models/service.ex (hunk 4 above)
Comment (krummi) [inline web/models/service.ex]: True :(

Comment (krummi) [inline web/models/service.ex]: Stupid `:greater_than_or_equal_to`... Why not `validate_number(:cleanup_duration, 0..60)`?

File: test/controllers/service_controller_test.exs (hunk 3 above)
Comment (krummi) [inline test/controllers/service_controller_test.exs:51]: have to do it in the next PR which is branched from this one - have already refactored some of this stuff there

**Pull request merged**